vanilla-framework/vanilla-framework
# 1505: Add overflow: hidden to icons

## Done

- Added `overflow: hidden` to the base icon pattern. This way, if text is added to an icon tag for accessibility, for example:
```html
<i class="p-icon--delete">delete</i>
```
it does not affect the icon styling in any way.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-light/
- Add text to some icons
- Check that nothing changes

## Details

Fixes #1505
